tracer/src/Datadog.Trace/Iast/SourceTypeUtils.cs

```diff
@@ -0,0 +1,72 @@
+// <copyright file="SourceTypeUtils.cs" company="Datadog">
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
+// </copyright>
+
+#nullable enable
+
+namespace Datadog.Trace.Iast;
+
+internal static class SourceTypeUtils
+{
+    // Defined in https://github.com/DataDog/experimental/blob/main/teams/asm/vulnerability_schema/vulnerability_schema.json
+
+    internal static string GetString(byte value)
+    {
+        return GetString((SourceType)value);
+    }
+
+    internal static string GetAsTag(SourceType value)
+    => value switch
+    {
+        SourceType.RequestParameterValue => "http_request_parameter",
```

Comment:
maybe these could be constant strings that you'd reuse

Reply:
Being a get only property, I wonder if compiler optimizes this